README.md

```diff
@@ -9,23 +9,159 @@ An engineer working for a [CSE](./CSE.md) project...
+3. Extensible heirarchy to allow teams to share deep subject matter expertise.
+
+### Working Heirarchy
+
+The working heirarchy is rooted in CSE's one week sprint cycle. Each section within then allows for deep dive information, recipes, and use cases.
+
+- [Project Start](0000-pre-sprint)
+  - [Team Agreements](0000-pre-sprint/team-agreements)
+    - [Working Agreements](0000-pre-sprint/team-agreements/working-agreements)
+    - [Definition of Ready](0000-pre-sprint/team-agreements/definition-of-ready)
+    - [Definition of Done](0000-pre-sprint/team-agreements/definition-of-done)
+    - [Estimation ](0000-pre-sprint/team-agreements/estimation)
+  - [Repository organization strategies (How many repositories? How should I decide?)](0000-pre-sprint/source-control-repositories)
+    - Setting up a new repository (readme, license, ignores, etc)
+      - Recipe for ADO
+      - Recipe for GitHub
+  - [Versioning (extend)](0000-pre-sprint/versioning)
+    - Recipe for implementing Semantic Versioning
+      - ADO Pipelines
+      - Jenkins
+  - [Building a Product Backlog](0000-pre-sprint/product-backlog)
+    - Guide to creating stories
+      - INVEST
+      - User story and acceptance criteria examples
+      - Defining stories for ML
+    - Recipes
+      - Managing product backlog in ADO
+- [Day 1](0010-day-one)
+  - [Sprint Planning](0010-day-one/sprint-planning)
+    - Purpose, Goals, Participants, Facilitation Guidance, Impact, and Measures
+    - Capacity Planning
+    - Tasking
+    - Dividing work WIP Limits
+  - [Test-First Development](0010-day-one/test-first-development)
+    - Conceptual (Purpose, Goals, Impact, and Measures)
+    - Developing Test Cases
+    - Unit Testing
+      - Conceptual (Purpose, Goals, Impact, and Measures)
+    - Load Testing
+  - [Feature Branching (creating branch for new story)](0010-day-one/feature-branching)
+- Day 2
+  - Commit best practices (move some existing content here)
+    - Link work items
+    - How often to commit
+    - When to push
+  - Continuous Integration (extend)
+    - Conceptual (Purpose, Goals, Impact, and Measures)
+      - Recipes for ADO
+  - Scrum of Scrums
+    - Purpose, Goals, Participants, Facilitation Guidance, Impact, and Measures
+  - Daily Standups
+    - Purpose, Goals, Participants, Facilitation Guidance, Impact, and Measures
+      - What should be in my standup update
+    - Recipes
+      - How to run efficient standups for remote teams
+- Day 3
+  - Pull Requests (separate from code reviews)
+    - Conceptual requirements for pull request (it should build, have 1 reviewer, linked work item, build changes)
+      - Add emphasis on importance of protecting master, effect this has on crew efficiency
+    - Recipe for Setup in
+      - Azure DevOps
+      - GitHub
+    - Code Reviews
+      - Conceptual
+        - Add to checklist (breaking changes & backward compatibility, security, fault tolerance, etc)
+  - Code Merging
```

Comment:
On the topic of VC systems, I suppose the expectation is that partners will always use Git. Is that a safe assumption?

I have a lot of thoughts and experience on code merging best practice (as well as the pros and cons of different policy restrictions in GH and ADO), so I'd like to officially volunteer to fill out that content.

Reply:
I think we pretty much have to assume that because the workflow for centralized VC systems (i.e TFS VC) is much different. I added an assumptions section at the bottom of the readme that indicates this assumption.